pages/services/edge-lb/0.1.9/uninstalling/index.md

```diff
@@ -8,6 +8,8 @@ excerpt:
 enterprise: false
 ---
 
+Uninstall the Edge-LB package with these instructions.
+
 *Do not* use the GUI or CLI (Marathon) to create or destroy services managed by Edge-LB. Operations must be performed exclusively through the Edge-LB CLI.
```

Comment:
Does this "*Do not...* section appear in the box on the landing page?

Reply:
No, only the first normal text before a blank line will appear in the boxes.